README.md

```diff
@@ -8,12 +8,38 @@ Note: `Kokkos` must be configured with `Kokkos_ENABLE_LIBDL=ON` to load profilin
 
 ## General Usage
 
-To use one of the tools you have to compile it, which will generate a dynamic library. Before executing the Kokkos application you then have to set the environment variable `KOKKOS_TOOLS_LIBS` to point to the dynamic library e.g. in the `bash` shell:
-```
-export KOKKOS_TOOLS_LIBS=${HOME}/kokkos-tools/src/tools/memory-events/kp_memory_event.so
-```
+To use one of the tools you have to compile it, which will generate a dynamic library. Before executing the Kokkos application you then have to set the environment variable `KOKKOS_TOOLS_LIBS` to point to the dynamic library. Many of the tools will produce an output file that uses the hostname as well as the process id as part of the filename.
+
+Use the route of either CMake or Makefile to build and run Kokkos Tools. The following provides instructions on both routes. 
+
+## Using cmake 
+
+### Build
+
+1. Create a build directory in Kokkos Tools, e.g., type `mkdir myBuild; cd myBuild`
+2. To configure, type `ccmake .. -DCMAKE_INSTALL_PREFIX=${YOUR_KOKKOS_TOOLS_INSTALL_DIR}` for any options you would like to enable/disable. 
```

Comment:
```suggestion
2. To configure, type `cmake .. -DCMAKE_INSTALL_PREFIX=${YOUR_KOKKOS_TOOLS_INSTALL_DIR}`. There aremore options but in most cases the defaults are sufficient.
```

Reply:
Thanks. If @dalg24 is OK with this, I am fine with it. See discussion on ccmake vs cmake above. Personally, I have come to find ccmake very useful for development but I understand it is non-intuitive for Kokkos Tools users.

Reply:
Also, FYI: my thought was that by mentioning ccmake provides a path for the user to see all the options available (and see that there are more options).

I agree with @dalg24 we should discuss this particular in another PR, which is a broader revamp of the README.md for style and these specifics.

This PR is essentially making sure all the basic build and run instructions that exist are correct and there are no inconsistencies (or outdated aspects) on it.